pages/your_domain/structural_bioinformatics.md

```diff
@@ -48,3 +48,19 @@ Researchers in the field should be able to find predictions of macromolecular st
   * input data needed for the modelling steps. For instance in homology modelling this could include the {% tool "pdb" %} identifiers for the template structures used for modelling and their alignments to the target protein;
   * model quality estimates.
 * If necessary, accompanying data can be provided in separate files using different file formats. The files can be added to ModelArchive depositions and referred to in the PDBx/mmCIF ModelArchive extension format.
+
+## Storing and sharing experimentally solved structures
+ 
+### Description
+
+Experimentally solved atomic structures of molecules can be obtained by several methods, such as X-ray crystallography,  Nuclear Magnetic Resonance (NMR) spectroscopy, and 3D Electron Microscopy. Here you can find useful tools and guides for storing and sharing structure models based on these methods.
+* Structure models resulting from experimental methods are broadly in the {% tool "pdb" %} under the [CC0 1.0 Universal (CC0 1.0) Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/).
+* Additionally, raw and intermediate data associated with the structure model can be also published in different curated data archives depending on the methods used. Raw EM data and processed 3D volumes and tomograms can be stored in {% tool "empiar" %} and {% tool "emdb" %}, respectively. Raw data from NMR studies can be stored in {% tool "bmrb" %}. For X-ray diffraction experiments raw data can be stored in {% tool "irrmc" %} and {% tool "sbgrid-data-bank" %}. Data submitted in these repositories can be cross-referenced to related PDB entries.
```

Comment:
>Data submitted in these repositories

Probably "to" is the right preposition here. Alternative, "Data stored in..."

Reply:
Done!